CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 15.6.0
+
+_To be released_
+
+### cli
+
+- :scroll: Document `allowGetBody` option in `z.request()` ([#752](https://github.com/zapier/zapier-platform/pull/752))
+
+### core
+
+- :wrench: Change trigger output check logic for the newly-added `primary` property in `outputFields` ([#754](https://github.com/zapier/zapier-platform/pull/754))
+
+### schema
+
+- :tada: Allow to set `primary` in `outputFields` to define the unique key for [deduplication](https://github.com/zapier/zapier-platform/blob/main/packages/cli/README.md#how-does-deduplication-work) ([#754](https://github.com/zapier/zapier-platform/pull/754))
+- :test_tube: Add support for throttle configuration override ([#755](https://github.com/zapier/zapier-platform/pull/754))
```

Comment:
This links to #754 despite the PR name states 755

Reply:
Fixed. Thanks!